src/ether_nethub.c

```diff
@@ -38,6 +38,12 @@
 #include "timerdefs.h"
 #include "lisp2cdefs.h"
 
+/* Needed for Alpine Linux */
+#include <sys/select.h>
```

Comment:
Could you say which symbols these are needed for?

Reply:
<sys/select.h> has defs for fd_set and FD_SET()
<sys/types.h> has typedef for u_char

I think its Posix standard for fd_set and FD_SET to be defined in sys/select.h, but a lot of platforms also define then elsewhere (like types.h).  Not so for Alpine.

And u_char is also not Posix but a BSD thing that most platforms define in various header files.  But on Alpine, it requires loading the sys/types.h header file.

I think all this stems from Alpine (and several other distros) using the musl package for libc instead of the glibc package, which is found on most of the "bigger" Linux (and apparently Mac) platforms.

Reply:
OK.  I'm just installing an Alpine Linux VM so I can test maiko things.

I'd like to move the code as close to POSIX as possible, while still compiling on things like Android, FreeBSD, other *BSD, Solaris 11/SPARC, MacOS/PPC, etc.